prime-router/src/test/kotlin/metadata/MapperTests.kt

```diff
@@ -685,6 +685,60 @@ class MapperTests {
         assertThat(actual.value).isEqualTo(expected)
     }
 
+    @Test
+    fun `test zip code to state mapper`() {
+        val mapper = ZipCodeToStateMapper()
+        val csv = """
+            zipcode,state_abbr
+            32303,FL
+            42223,TN
+            42223,KY
```

Comment:
Maybe add some 5-4 zip codes (e.g. `32303-1234`) since code should handle them?

Reply:
good calll here too. Will do